src/components/sidebar/index.ts

```diff
@@ -0,0 +1 @@
+export{default as Sidebar} from "./Sidebar";
```

Comment:
dodaj dodatkową linię na końcu. Używaj sobie prettiera najlepiej, który powinien to robić za Ciebie